lib/composer/wikia/fluent-sql-php/tests/FluentSql/NonNumericLimitOffsetTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Test non-numeric LIMIT and OFFSET
+ *
+ * @author Mix <mix@wikia.com>
+ */
+
+namespace FluentSql;
+
+class NonNumericLimitOffsetTest extends FluentSqlTestBase {
+
+	/**
+	 * numeric LIMIT
+	 */
+	public function test1() {
```

Comment:
Is this test naming a convention in the lib? Maybe those could be more descriptive?